docs/monitoring/index.asciidoc

```diff
@@ -5,19 +5,23 @@
 [partintro]
 --
 
-{monitoring} in {kib} serves two separate purposes:
+The {kib} {monitor-features} serve two separate purposes:
 
-. To visualize monitoring data from across the Elastic Stack. You can view
-health and performance data for {es}, Logstash, and Beats in real time, as well
-as analyze past performance. For more information, see {xpack-ref}/xpack-monitoring.html[Monitoring the Elastic Stack].
+. To visualize monitoring data from across the {stack}. You can view health and 
+performance data for {es}, {ls}, and Beats in real time, as well as analyze past 
+performance. For more information, see 
+{stack-ov}/xpack-monitoring.html[Monitoring the {stack}].
 
 . To monitor {kib} itself and route that data to the monitoring cluster.
 
-If you enable {monitoring} across the Elastic Stack, a monitoring agent runs on
-each {es} node, Logstash node, {kib} instance, and Beat to collect and index
+If you enable monitoring across the {stack}, a monitoring agent runs on
+each {es} node, {ls} node, {kib} instance, and Beat to collect and index
 metrics. Each node and instance is considered unique based on its persistent
 UUID, which is written to the <<settings,`path.data`>> directory when the node
-or instance starts.
+or instance starts. 
+
+beta[] Alternatively, in 6.4 and later versions, you can use {metricbeat} to 
+collect and ship data about {kib} and {es}. 
```

Comment:
So, technically, one could use Metricbeat to collect and ship data about Kibana starting from 6.4 and about Elasticsearch from 6.5.

Reply:
Good catch, @ycombinator!  I decided that information was not required in this introductory page, so I've removed it.